python/cli/prelude_cli/views/iam.py

```diff
@@ -230,52 +256,13 @@ def logs(controller, days, limit):
         return controller.audit_logs(days=days, limit=limit)
 
 
-@iam.command("subscribe")
-@click.argument(
-    "event",
-    type=click.Choice(
-        [e.name for e in AuditEvent if e != AuditEvent.INVALID], case_sensitive=False
-    ),
-)
+@iam.command("new-user-and-account", hidden=True)
```

Comment:
wanna just call this sign-up. b/c i'm too lazy to type and also it's what FE calls it